src/Symfony/Component/Cache/DependencyInjection/CachePoolPass.php

```diff
@@ -181,11 +181,11 @@ public function process(ContainerBuilder $container)
             $container->removeDefinition('cache.early_expiration_handler');
         }
 
-        $notAliasedCacheClearerId = $aliasedCacheClearerId = 'cache.global_clearer';
-        while ($container->hasAlias('cache.global_clearer')) {
-            $aliasedCacheClearerId = (string) $container->getAlias('cache.global_clearer');
+        $notAliasedCacheClearerId = 'cache.global_clearer';
+        while ($container->hasAlias($notAliasedCacheClearerId)) {
+            $notAliasedCacheClearerId = (string) $container->getAlias($notAliasedCacheClearerId);
```

Comment:
The `$clearers` array should only use non-alias names since `$container->getDefinition()` (on line 193) doesn't work with aliases.